tasks.go

```diff
@@ -56,6 +62,32 @@ func GetAllTasks(basepath string) ([]Task, error) {
 	return tasks, nil
 }
 
+// GetNagiosDataTask will return an array of Tasks each of which will dump the nagios data for one project.
+func GetNagiosTasks(projects []string, basepath string) ([]Task, error) {
+	var tasks []Task
+	for _, p := range projects {
+		pods, err := getResourceNamesBySubstr(p, "pod", "nagios")
+		if err != nil {
+			continue
+		}
+		if len(pods) == 0 {
+			continue
+		}
+		for _, pod := range pods {
+			outFor := outToFile(basepath, "dat", "logs")
```

Comment:
I felt like the status and historical data of Nagios was like a log of what nagios has done. What would you recommend here, a new folder like "Monitoring"?